### ISSUE TITLE
Epic: post-merge content updates

### ISSUE THREAD

**Issue by jmcook1186** [ethereum/ethereum-org-website#7075]
**Epic Description**

The Merge is almost upon us. Ethereum.org has recently been updated to ensure that the content is up-to-date and correct for the final months of pre-merge Ethereum. Now, we will create a branch of ethereum.org with content appropriate for post-merge Ethereum, with the intention to merge this branch in once The Merge is complete.

**Involves**

- Scoping out what content changes, additions, or subtractions are required related to the Merge upgrade (done, tracked in Excel sheet)
- Preparing appropriate content changes,
- Coordinating with external contributors to ensure accuracy
- Timing releases appropriately to provide the community with accurate information about the Merge during and immediately after transition

**Goals**

- Ensure references to proof-of-work are updated to either be historical references, or removed
- Update developer documentation to ensure all fundamentals are accurately represented under the proof-of-stake model
- Update /upgrade/merge and to be reflective of past tense and latest information
- Update /energy-consumption page with updated latest information

**Checkpoints**

[Page Audit Spreadsheet](https://docs.google.com/spreadsheets/d/1Nisi3_FY8nwOlLwPsTPm0jlgtAhypH0-pjatGDMSz7M/edit#gid=0)

The following pages have been identified as requiring post-merge adjustments (tick off when PR merge to `post-merge` branch):

- [ ] /nft/
- [ ] /developers/docs/gas/
- [ ] /developers/docs/consensus-mechanisms/pow/mining/
- [ ] /staking/
- [ ] /developers/
- [ ] /upgrades/
- [ ] /upgrades/merge/
- [ ] /developers/docs/web2-vs-web3/
- [ ] /developers/docs/consensus-mechanisms/pos/
- [ ] /developers/docs/intro-to-ethereum/
- [ ] /developers/docs/evm/
- [ ] /developers/docs/nodes-and-clients/
- [ ] /learn/
- [ ] /eth/
- [ ] /developers/docs/accounts/
- [ ] /developers/docs/dapps/
- [ ] /upgrades/beacon-chain/
- [ ] /developers/docs/transactions/
- [ ] /developers/docs/networks/
- [ ] /developers/docs/consensus-mechanisms/pow/
- [ ] /developers/docs/mev/
- [ ] /developers/docs/blocks/
- [ ] /developers/docs/consensus-mechanisms/
- [ ] /developers/docs/intro-to-ether/
- [ ] /upgrades/shard-chains/
- [ ] /glossary/
- [ ] /developers/docs/apis/json-rpc/
- [ ] /developers/docs/apis/backend/
- [ ] /energy-consumption/
- [ ] /security/
- [ ] /developers/docs/storage/
- [ ] /community/online/
- [ ] /developers/docs/apis/javascript/
- [ ] /upgrades/vision/
- [ ] /developers/docs/data-and-analytics/block-explorers/
- [ ] /developers/docs/nodes-and-clients/nodes-as-a-service/
- [ ] /developers/tutorials/uniswap-v2-annotated-code/
- [ ] /governance/
- [ ] /developers/tutorials/run-light-node-geth/
- [ ] /developers/docs/development-networks/
- [ ] /developers/docs/data-and-analytics/
- [ ] /developers/tutorials/run-node-raspberry-pi/
- [ ] /developers/tutorials/a-developers-guide-to-ethereum-part-one/
- [ ] /community/support/
- [ ] /developers/tutorials/hello-world-smart-contract-fullstack/
- [ ] /developers/docs/nodes-and-clients/client-diversity/
- [ ] /developers/tutorials/downsizing-contracts-to-fight-the-contract-size-limit/
- [ ] /developers/tutorials/learn-foundational-ethereum-topics-with-sql/
- [ ] /deprecated-software/
- [ ] /developers/tutorials/using-websockets/
- [ ] /developers/docs/consensus-mechanisms/pow/mining-algorithms/
- [ ] /developers/docs/networking-layer/
- [ ] /developers/docs/mining/
- [ ] /developers/docs/consensus-mechanisms/pow/mining-algorithms/ethash

**Comment by minimalsm** [comment]
Just noting [in this commit](https://github.com/ethereum/ethereum-org-website/pull/7264/commits/3173f7351514bfed28ee18db4b3bf353b01581b0), I anchored some content to The Merge happening in September. If it doesn't happen in September, we should fix this ASAP.

**Comment by samajammin** [comment]
FYI @jmcook1186 @wackerow updated the issue description to include the ` /upgrades/merge/issuance/` page.

**Comment by wackerow** [comment]
Sounds good. A community member also reported that the [Ethereum Vision](https://ethereum.org/en/upgrades/vision/) page also still notes "Too much energy" as one of the problems facing Ethereum today:
<img width="1164" alt="image" src="https://user-images.githubusercontent.com/54227730/190649342-2c1aec8d-237b-48be-94bd-309e08127b5c.png">

I think we should update this. Few options include:
1. Change the title to be more broadly the issues that are being addressed, and update the description for that third card to be more of a reflection
2. Remove this card
3. Replace the card with a new issue that challenges the scalability of Ethereum
Open to suggestions

**Comment by samajammin** [comment]
> A community member also reported that the [Ethereum Vision](https://ethereum.org/en/upgrades/vision/) page also still notes "Too much energy" as one of the problems facing Ethereum today:

Yes great point!

I'm open to any ideas. I like the idea of (1), to change the section title (maybe something like "Problems being addressed"). Then in each card description, we could link to resources on how the problem is being tackled:

- Clogged network: mention roll-up centric roadmap & link to L2 page?
- Disk space: linking to sharding / research resources on state expiry?
- Energy consumption: mention this is largely solved, link to Merge/PoS page?

**Comment by ChXmusuke** [comment]
> A community member also reported that the [Ethereum Vision](https://ethereum.org/en/upgrades/vision/) page also still notes "Too much energy" as one of the problems facing Ethereum today:
>

i also like solution (1), maybe alongside adding a green frame to the tile and a "solved since the Merge" text going with it?

**Comment by samajammin** [comment]
Hey @jmcook1186 @wackerow - is this complete? Can this be closed out?
